notebooks/starter_notebook.py

```
@@ -0,0 +1,260 @@
+
+    Returns:
+    DataFrame: A new DataFrame with the specified columns removed.
+    """
+    return dataframe.drop(columns=column_names)
+
+# COMMAND ----------
+
+def separate_features_and_target(dataframe: pd.DataFrame, target_column: str) -> (pd.DataFrame, pd.DataFrame):
+    """
+    Separates the features and the target column for modeling.
+    
```

Ruff

Blank line contains whitespace

(W293)